apps/perf-test/src/Measurer.tsx

```diff
@@ -27,6 +30,8 @@ export class Measurer extends React.Component<{}, {}> {
 
     Measurer._timings = _createTimings();
 
+    console.log(JSON.stringify(timings));
```

Comment:
probably should skip the console.log in the component

Reply:
Will revert all changes to old app.. was an initial approach that I've since ditched for flamegraphs.